package.json

```diff
@@ -48,11 +48,11 @@
     "rollup-plugin-postcss": "^4.0.1",
     "rollup-plugin-vue": "^5.1.9",
     "vue": "^2.6.14",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.7.8"
   },
   "peerDependencies": {
     "@nextcloud/l10n": "^1.4",
-    "@nextcloud/vue": "^4.2||^5.0",
+    "@nextcloud/vue": "^6.0.0-beta.1",
     "vue": "^2.6"
```

Comment:
Needs a bump as well

Reply:
are you referring to updating vue ? I saw on server we also have vue 2.6, so probably need to move it up there as well